pipeline/coordinator.go

```diff
@@ -28,16 +28,12 @@ import (
 type Strategy string
 
 const (
-	// Only execute the Catalyst (Mist) pipeline.
-	StrategyCatalystDominance Strategy = "catalyst"
-	// Only execute the external pipeline.
+	// Only execute the external pipeline
 	StrategyExternalDominance Strategy = "external"
 	// Only execute the FFMPEG / Livepeer pipeline
 	StrategyCatalystFfmpegDominance Strategy = "catalyst_ffmpeg"
```

Comment:
I'm not sure we need to have `catalyst_ffmpeg`, maybe we should just keep the old name `catalyst`?

If you decide to keep `catalyst_ffmpeg`, then please update the related configs here: https://github.com/livepeer/catalyst/tree/main/config